pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -462,6 +462,8 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		if vmi.Spec.Domain.CPU.Model != "" {
 			domain.Spec.CPU.Mode = "custom"
 			domain.Spec.CPU.Model = vmi.Spec.Domain.CPU.Model
+		} else {
```

Comment:
Can we extract the model logic and do it after the user-focused customizations? Something like:

```golang
if vme.Spec.Dmonain.CPU != nil {
 # process user requests 
}

# then afterwards
if vmi.Spec.Domain.CPU == nil || vmi.Spec.Domain.CPU.Model == "" {
  domain.Spec.CPU.Mode= "host-model"
}